cmd/vulcan-exposed-http-resources/README.md

```diff
@@ -0,0 +1,53 @@
+## Usage
+
+This check tests if a given web address exposes sensitive resources. The check tests a list of paths by performing an HTTP request with the GET method. There are two ways for specifying the list of resources to be tested:
+
+1. By default it will check if the resources listed in the `resources.yaml` file are exposed.
+
+2. If resources are specified through options, the check will instead look for those resources.
+
+The expected format for the options is:
+
+```
+{"resources":[
+	{"paths":["/one", "/two"], "status":200, "regex":"danger(ous)?", "severity":8.9},
+	{"paths":["/three"], "regex":"error.*found", "description":"Exposed error file."}
+]}
+```
+
+## Behaviour
+
+For each resource to check for, a set of possible paths are specified, along with a status code and a regex to look for in the response.
+
+- The check will test each path for any given resource and report each one that is found.
+
+- If a status code is specified, the check will consider the resource to be exposed if the response to the request has the specified status code.
+
+- If a regular expression is specified, the check will consider the resource to be exposed if the whole HTTP response matches the regular expression. The regular expression must follow the syntax used by the Go `regexp` package.
+
+- If both conditions are specified, the check will consider the resource to be exposed if both of them are met.
+
+- If no conditions are specified the check will consider the resource to be exposed if the request returns any response.
+
+## Resources
+
+Resources that the check can identify as exposed are listed in the `resources.yaml` file. Any resource added to the file should have a short but clear description and should be accompanied preferably by a matching regular expression. If a reliable regular expression cannot be written, a status code should be used instead and the description should be preceded by the word "potentially"; the severity of the vulnerability should not be lowered for this reason alone.
+
+## Testing
```

Comment:
I like a lot the fact that there is an out of the box documented way to test resources.

Reply:
Thanks man! I wanted to make sure anyone would have an easy way to test contributions.